CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add support for `node_pool_auto_config {}` with `network_tags`
+
+### Changed
+
+- Upgraded provider to `~> 4.34` to add support for `node_pool_auto_config`
```

Comment:
missed we really actually change to `google-beta` provider here.. not really breaking but people need to configure and pin this provider now when using the module.. so maybe BREAKING again? ;) sorry for the confusion but the message is also missing the change here ;)